go.mod

```diff
@@ -8,10 +8,13 @@ go 1.13
 
 require (
 	bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
-	github.com/Azure/azure-sdk-for-go v35.0.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.9.0
-	github.com/Azure/go-autorest/autorest/adal v0.5.0
-	github.com/Azure/go-autorest/autorest/to v0.2.0
+	github.com/Azure/azure-pipeline-go v0.2.2 // indirect
```

Comment:
do we need to update the go modules or is just an "accident"?

Reply:
I run these tests locally, and compilation updated the `go.mod` file, too. I did not introduce new dependency.

Reply:
if you didn't intend to change this, then revert the changes before committing and pushing

Reply:
Fixed